apps/backend/src/mutations/ProposalMutations.ts

```diff
@@ -517,6 +516,165 @@ export default class ProposalMutations {
     return result || rejection('Can not change proposal status', { result });
   }
 
+  @EventBus(Event.PROPOSAL_STATUS_CHANGED_BY_USER)
+  @Authorized([Roles.USER_OFFICER])
+  async changeProposalsStatus(

```

Comment:
Git shows this as a new addition, but this is unchanged from before and the resolver still exists. The implementation for it is now moved to a private function `processProposalsStatusChange` so that both this mutation `changeProposalsStatus` and the Xpress one `changeXpressProposalsStatus` can use it without duplication.

https://github.com/UserOfficeProject/user-office-core/blob/012587b2641ac4757f061d983073c8d9045a65c6/apps/backend/src/resolvers/mutations/ChangeProposalsStatusMutation.ts#L26

Reply:
Noted. Thanks.